doc/WebRTC.xml

```diff
@@ -829,8 +829,12 @@ Server -> Client: { "error": {"code": 1001, "message": "Insufficient resources"}
       <title>Audio</title>
       <para>When using WebRTC with a web browser only certain codecs will work. Because of this
         ONVIF recommends to use the Opus or PCMU codec.</para>
-      <para>For two-way audio it's recommended to use echo cancellation to avoid feedback
-        loops.</para>
+      <para>If the selected media profile contains an AudioDecoder, the device shall include an
+        audio track with <literal>a=recvonly</literal> in its SDP offer. If the selected media profile contains
+        an AudioEncoder, the device shall include an audio track with <literal>a=sendonly in its SDP offer</literal>.
+        If the profile contains both AudioEncoder and AudioDecoder, the device shall include an audio
+        track with <literal>a=sendrecv</literal> in its SDP offer. For two-way audio it is  recommended to use echo 

```

Comment:
Should echo cancellation be mandated for RTP sessions when both AudioEncoder and AudioDecoder are supported? It may be preferable to clarify that such an RTP session is considered additional and optional in this case, rather than mandatory.